application-collabora-default/src/main/java/com/xwiki/collabora/internal/FileToken.java

```diff
@@ -54,6 +54,8 @@ public class FileToken
 
     private int usage;
 
+    private String lastActionMode;
```

Comment:
I would simply name it `mode` or `actionMode`
So your methods would be for example `setMode` or `setActionMode` 

Also, you could use an Enum